components/Flipper/FlipRestrictionList/FlipRestrictionList.tsx

```diff
@@ -396,8 +413,21 @@ function FlipRestrictionList(props: Props) {
                 )}
                 <hr />
             </div>
+            <Form.Control style={{ width: '49%', marginLeft: '20px' }} placeholder="Search..." onChange={e => setSearchText(e.target.value)} />
             <div className={styles.restrictionList}>
                 {restrictions.map((restriction, index) => {
+                    if (searchText) {
+                        let isValid = false
+                        if (restriction.item?.name && restriction.item?.name.toLowerCase().includes(searchText.toLowerCase())) {
+                            isValid = true
+                        }
+                        if (restriction.tags && restriction.tags.includes(searchText)) {
```

Comment:
This only allows for exact matches of the tag. Consider supporting partial matches of tag text